userguide/tutorials/what_is_kill_bill.adoc

```diff
@@ -2,26 +2,30 @@
 
 == Introduction
 
-Kill Bill is a mature open-source platform which powers the billing and payment infrastructure of some of the largest e-commerce websites.
+Kill Bill is an open-source programmable platform that enables you to build custom billing solutions. That's a mouthful, to say the least, but think of it as building blocks to implement your own billing workflows.
 
-The *subscription billing* features let you create recurring or usage based subscriptions (e.g. "gold plan subscription at $9.99/month") and manage upgrades and downgrades. The system will take care of generating invoices, managing credits and triggering payments. Netflix, Dropbox and The New York Times are examples of websites which sell such subscriptions.
+Similar to the Unix philosophy of composability over monolithic design, Kill Bill provides APIs to create various pricing models, manage subscriptions, generate invoices, integrate with various payment gateways, etc. Whether you are selling access to a digital service for $10 a month or selling a multi-tier cell-phone data plan, Kill Bill provides the primitives to do it so you don't have to start from scratch.
```

Comment:
```suggestion
Kill Bill APIs let you create various pricing models, manage subscriptions, generate invoices, integrate with various payment gateways, etc. Whether you are selling access to a digital service for $10 a month or selling a multi-tier cell phone data plan, Kill Bill provides the primitives to do it so you don't have to start from scratch.
```

Reply:
The Unix philosophy bit would speak to the engineering audience. Do you feel strongly against it?